roles/edpm_ovn/tasks/install.yml

```diff
@@ -21,7 +21,6 @@
     setype: "container_file_t"
     mode: "{{ item.mode | default(omit) }}"
   loop:
-    - {'path': "{{ edpm_ovn_neutron_metadata_agent_config_dir }}"}
     - {'path': /var/log/containers/openvswitch, 'mode': '0750'}
     - {'path': /var/log/containers/neutron, 'mode': '0750'}
```

Comment:
do we need this neutron log dir here anymore?

Reply:
(For some reason, github UI doesn't allow me to comment in-place) below, you install cleanup script for neutron. Should it be moved too?

Reply:
Not sure I get this 2 comments.
For the first one, that one I removed it from ovn-controller and added it on metadata. Are you saying this is still needed on the ovn-controller side?

And for the second one, the scripts were there before and that is about configuring the ovs bridge (in the same way we configure its external-ids). Are you asking if this should still be there (perhaps a different PR anyway)?

Reply:
For the first one, I wonder why edpm_ovn role - still - creates a directory with `neutron` in its name (for logs). I don't think it should.

For the second, I may miss the goal of neurtron-cleanup script. (I triggered on the name, but maybe it's still ovn specific?) I will need to check, for now ignore.

Reply:
right!